### PR TITLE
react-addons-perf: Update APIs to version 15.1.0

### PR DESCRIPTION
**Pull request by rhysd**
case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url https://facebook.github.io/react/docs/perf.html.
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: react/react-addons-perf.d.ts [hunk 1]
```diff
@@ -31,10 +31,11 @@ declare namespace __React {
...
             export function stop(): void;
-            export function printInclusive(measurements: Measurements[]): void;
-            export function printExclusive(measurements: Measurements[]): void;
-            export function printWasted(measurements: Measurements[]): void;
-            export function printDOM(measurements: Measurements[]): void;
+            export function printInclusive(measurements?: Measurements[]): void;
+            export function printExclusive(measurements?: Measurements[]): void;
+            export function printWasted(measurements?: Measurements[]): void;
+            export function printOperations(measurements?: Measurements[]): void;
+            export function printDOM(measurements?: Measurements[]): void;
```
Comment (vsiao) [inline react/react-addons-perf.d.ts]: could you add a comment mentioning that this is deprecated?

**Comment by vsiao** [comment]
thanks, looks good.

File: react/react-addons-perf.d.ts (hunk 1 above)
Comment (rhysd) [inline react/react-addons-perf.d.ts]: I will do :+1:

**Comment by rhysd** [comment]
@vsiao I added deprecation comment to `printDOM()`.

**Comment by rhysd** [comment]
I added more `get***` methods.  They are useful to create the perf result as JSON.

**Pull request merged**